parsing/lexer.mll

```diff
@@ -349,6 +352,8 @@ rule token = parse
           NATIVEINT (cvt_nativeint_literal (Lexing.lexeme lexbuf))
         with Failure _ ->
           raise (Error(Literal_overflow "nativeint", Location.curr lexbuf)) }
+  | (float_literal | int_literal) identchar*
```

Comment:
What's the point of the star here? You want to raise as soon as you see one `identchar`, and you don't want to raise if there are none.